ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -453,10 +453,16 @@ objects:
         mkdir -p ~/.ssh
         cp "${SSH_PRIV_KEY_PATH}" ~/.ssh/
 
+        masters=3
+        if has_variant "sno" ;then
```

Comment:
Let's make the variant `single-node`.  `sno` doesn't mean anything and we have characters to spare.

Reply:
OK, done